tenant/vendor/sigs.k8s.io/controller-tools/testData/config/default/kustomization.yaml

```diff
@@ -21,5 +21,5 @@ resources:
 - ../rbac/*.yaml
 - ../manager/*.yaml
 
-patches:
+patchesStrategicMerge:
```

Comment:
In general, you should never make a change inside a `/vendor` directory in Go. `/vendor` is the old way of importing dependencies (the new way is called [modules](https://blog.golang.org/using-go-modules)) and they should always reflect the original source code as much as possible.

In this case, it's also unnecessary - the only point to putting something in `/vendor` is to let the Go compiler find `.go` files, so no-one will ever run `kustomize` in that directory too.

Reply:
:man_facepalming: I'm, sorry. I did a `sed` on my repo and this passed my eyes. I'll revert it.